src/strsearch_enc_1.cpp

```diff
@@ -253,43 +251,33 @@ int main()
   int p = 786433; //plaintext prime modulus
   //int p = 65537; //note this causes exception
 
-  cout<<"p "<<p<<endl;
+  cout << "p " << p << endl;
   TIC(auto t1);
 
   auto presult = search(pat, txt, p);
   auto plain_time_ms = TOC_MS(t1);
-  cout<< "Plaintext execution time "<<plain_time_ms<<" mSec."<<endl;
+  cout << "Plaintext execution time " << plain_time_ms << " mSec." << endl;
 
-  cout <<"setting up BFV RNS crypto system"<<endl;
+  cout << "setting up BFV RNS crypto system" << endl;
 
   uint32_t plaintextModulus = p;
   uint32_t multDepth = 32;
 
-  double sigma = 3.2;
   lbcrypto::SecurityLevel securityLevel = lbcrypto::HEStd_128_classic;
 
-    /**
```

Comment:
NOTE:

Could it be that we have different defaults? I noticed that I'm not using `sigma` or `numAdds` or keyswitches or whatever.